src/rail_gui.cpp

```diff
@@ -1690,10 +1693,25 @@ struct BuildSignalWindow : public PickerWindowBase {
 				y + this->IsWidgetLowered(widget_index));
 	}
 
+	void SetSignalUIMode() {
```

Comment:
```suggestion
	void SetSignalUIMode()
	{
```

Could do with some doxygen too